net-libs/libsrtp/libsrtp-1.6.0-r1.ebuild

```diff
@@ -45,6 +45,9 @@ src_prepare() {
 
 	mv configure.in configure.ac || die
 	eautoreconf
+
+	# Compatibility with sys-devel/autoconf:2.71
+	sed -i -e 's|install-sh ar-lib|install-sh|' configure
```

Comment:
Can you use the approach from 4710a539a8237cd8afe2bfcc92c2e0850ce521ff? I want this pushed today, so sorry about the fast notice.

Reply:
Sure, I'll check if that also solves the issue and change this PR then. Thanks for the hint

Reply:
@juippis : Done. Fix looks better that way